website/docs/advanced-concepts/access-control/granular-access-control/roles.md

```diff
@@ -0,0 +1,242 @@
+
+
+|  Permissions       |    Description                                                     |   Resource   |
+| ---------------    |------------------------------------------------------------------- | ------------ |
+| **Create**         | Enables permission to create, edit, view, and delete resources. You can also grant users access to create groups and roles. | [Application Resources](#application-resources-permissions), [Datasource and queries](#datasource-and-queries-permissions), [Groups and roles](#groups-and-roles-permissions), [others](#others-permissions). |
+| **Edit**           |Enables permission to edit and view resources. You can also grant users access to create groups and roles | [Application Resources](#application-resources-permissions), [Datasource and queries](#datasource-and-queries-permissions), [Groups and roles](#groups-and-roles-permissions), [others](#others-permissions). |
+| **View**           |Enables permission to only view resources. With the View permission, a user can only view the existing groups and roles | [Application Resources](#application-resources-permissions), [Datasource and queries](#datasource-and-queries-permissions), [Groups and roles](#groups-and-roles-permissions), [others](#others-permissions). |
+| **Delete**         |Enables permission to delete resources. You can also grant users access to delete groups and roles |[Application Resources](#application-resources-permissions), [Datasource and queries](#datasource-and-queries-permissions), [Groups and roles](#groups-and-roles-permissions), [others](#others-permissions). |  
+| **Export**         |Grants users permission to export applications in a workspace. | [Application Resources](#application-resources-permissions) | 
+| **Invite**         |Grants users permission to invite users to a user group | [Groups and roles](#groups-and-roles-permissions)     |
+| **Make public**    |Grants users permission to make the applications public in a workspace |[Application Resources](#application-resources-permissions)|
+| **Execute**        |Grants users permission to execute queries on a datasource | [Datasource and queries](#datasource-and-queries-permissions) |
+| **Remove users**   |Grants users permission to remove a user from a group| [Groups and roles](#groups-and-roles-permissions) |
+
+Let's understand these permissions in detail.
+
+### Application resources permissions
+
+On an application level, here are the permissions that can be assigned to user or a group -
+
+#### Create
+
+With this permission, users can create applications, pages, API queries and JS Objects. This also automatically enables permission to view, edit, and delete existing pages, API, queries and JS Objects. Under Application Resources, you can control the **Create** permission at different hierarchical levels:
```

Comment:
Since we discussed we won't write about the side-effects of a permission, should we remove `This also automatically enables permission to view, edit, and delete existing pages, API, queries and JS Objects.`?

Also, on line 36 it says `on application level`, shouldn't that be on workspace level? Or we can also write straight away `here are the permissions that can be assigned to user or a group -`